packages/aft/lib/src/models/package_selector.dart

```diff
@@ -1,12 +1,22 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+import 'dart:collection';
+
 import 'package:aft/aft.dart';
 import 'package:aws_common/aws_common.dart';
+import 'package:collection/collection.dart';
 import 'package:glob/glob.dart';
 import 'package:json_annotation/json_annotation.dart';
 import 'package:path/path.dart' as p;
 
+class _PathSet extends DelegatingSet<String> {
```

Comment:
Is this class the equivalent of the previous 'Set<String>.of` for removing duplicate entries in `Iterable<String> paths`?

Out of curiosity why did you switch away from `Set<String>.of`?  

Why can't we just use a HashSet directly since we're creating it and wrapping it with the DelegatingSet anyway?  Could we just declare a `Set` variable type and set it equal to the `HashSet`?

Reply:
Path comparison requires a special `==` method, which is why `Set.of` does not work. This delegates to a `HashSet` with `==` overridden correctly.